source/features/clean-issue-filters.tsx

```diff
@@ -0,0 +1,43 @@
+import select from 'select-dom';
+import features from '../libs/features';
+import {v4} from '../libs/api';
+import {getOwnerAndRepo, idx} from '../libs/utils';
+
+async function init(): Promise<void> {
+	const {ownerName, repoName} = getOwnerAndRepo();
+	const result = await v4(`
+		query {
+			repository(owner: "${ownerName}", name: "${repoName}") {
+				projects { totalCount }
+				milestones { totalCount }
+				labels { totalCount }
+			},
+			organization(login: "${ownerName}") {
+				projects { totalCount }
+			}
+		}
+	`, {
+		allowErrors: true
+	});
+
+	// Hide projects filter only if there are no repo and organization level projects
+	if (result.repository.projects.totalCount === 0 && !idx(result, ['organization', 'projects', 'totalCount'])) {
```

Comment:
Is idx required here? Maybe it’s just `results.organization && results.organization.projects.totalCount` because if organization exists, projects and their count exist too, I’d think.